tests/integration/search/hep/test_titles.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+from __future__ import absolute_import, division, print_function
+
+import json
+
+from test_utils import parametrize
+
+
+FIXTURES_FILE = 'hep_titles.xml.gz'
+
+
+def _query_and_retrieve_control_numbers(client, query_string):
+    response = client.get(
+        '/literature/?q=' + query_string,
+        headers={'Accept': 'application/vnd+inspire.ids+json'}
+    )
+
+    assert response.status_code == 200
+
+    response_json = json.loads(response.data)
+    return set([recid for recid in response_json['hits']['recids']])
+
+
+@parametrize(
+    {
+        'f t N-body and t separable': {
+            'query_str': 'f t N-body and t separable', 'expected_recids': {
+                # "The Relativistic N body problem in a separable two-body basis".
+                534466,
+                # 577837 title: "Reliability of the dark matter clustering in cosmological n-body simulation on scales
+                # below the mean separation length of particles".
+                # 313341 title: Exact n-d scattering calculation with a separable expansion of the two-body t matrix
+                # i.e. must not be there.
+            }
+        },
+        # TODO
+        # 't n body (should return both "n body" and "n-body")': {
+        #     'query_str': 't n body', 'expected_recids': {534466, 577837}
+        # },
```

Comment:
We are forced to do this ugly thing because your `parametrize` doesn't support this feature of pytest's `parametrize`: https://docs.pytest.org/en/latest/skipping.html#skip-xfail-with-parametrize. Why do we need your `parametrize` in the first place, and what can be done to recover this feature there?